### ISSUE TITLE
[Bug]: CONCAT_WS with null bug

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#3117]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):a557ce3
- Hardware parameters: M1 macOS arm version
- OS type:
- Others:
```

### Actual Behavior

[SQL STATEMENT]: select concat_ws(',','',NULL,'a');
[EXPECT RESULT]:
concat_ws(',','',NULL,'a')
,a
[ACTUAL RESULT]:
Function 'concat_ws' with parameters [VARCHAR VARCHAR unexpected type: 0 VARCHAR] will be implemented in future version.

### Steps to Reproduce

```markdown
select concat_ws(',','',NULL,'a');
```

**Comment by broccoliSpicy** [comment]
will be fixed by pr#3110

**Comment by broccoliSpicy** [comment]
fixed
